<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zookeepermanagewidget</class>
 <widget class="QWidget" name="zookeepermanagewidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget_left" native="true">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>OPPOSans B</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_left{
border-right:2px solid rgba(108, 117, 125, 65); 
}
QScrollBar:vertical{
    width: 10px;
	background-color: rgba(0, 41, 69, 0);
    margin: 0px,0px,0px,0px;
    padding-top: 0px;		/*//隐藏上下的箭头*/
    padding-bottom: 0px;
}


QScrollBar::handle:vertical{
    width: 10px;
	
	background-color: rgb(239, 239, 239);
    /*滚动条两端变成椭圆 */
    border-radius: 2px;
    min-height: 0;
}

QScrollBar::sub-page:vertical {
    
	background-color: rgba(255, 255, 255, 0);
}
QScrollBar::add-page:vertical 
{
    background-color: rgba(255, 255, 255, 0);
}

QScrollBar::add-line:vertical {
	border: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
    border: none;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::down-arrow:vertical {
    border:none;
}

QScrollBar::up-arrow:vertical {
    border:none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>9</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget_9" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>35</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>35</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QToolButton" name="toolButton_newCreate">
           <property name="minimumSize">
            <size>
             <width>180</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>OPPOSans B</family>
             <pointsize>10</pointsize>
             <stylestrategy>PreferAntialias</stylestrategy>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	
	
	background-color: rgb(20, 20, 20);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}
 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(20, 20, 20);
}</string>
           </property>
           <property name="text">
            <string>新建连接</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QScrollArea" name="scrollArea">
           <property name="styleSheet">
            <string notr="true">QScrollArea{
	border: none;
}

QScrollBar:vertical {
    width: 10px;
	background-color: rgba(0, 41, 69, 0);
    margin: 0px,0px,0px,0px;
    padding-top: 0px;		/*//隐藏上下的箭头*/
    padding-bottom: 0px;
}
QScrollBar::handle:vertical {
    width: 10px;
	
	background-color: rgb(239, 239, 239);
    /*滚动条两端变成椭圆 */
    border-radius: 2px;
    min-height: 0;
}

QScrollBar::sub-page:vertical {
    
	background-color: rgba(255, 255, 255, 0);
}
QScrollBar::add-page:vertical 
{
    background-color: rgba(255, 255, 255, 0);
}

QScrollBar::add-line:vertical {
	border: none;
    height: 0px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;
}
QScrollBar::sub-line:vertical {
    border: none;
    height: 0px;
    subcontrol-position: top;
    subcontrol-origin: margin;
}

QScrollBar::down-arrow:vertical {
    border:none;
}
QScrollBar::up-arrow:vertical {
    border:none;
}</string>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>186</width>
              <height>458</height>
             </rect>
            </property>
            <property name="styleSheet">
             <string notr="true">QToolButton {
	font: 10pt &quot;OPPOSans B&quot;;
    qproperty-iconSize: 22px 22px;
	background-color: rgba(255, 255, 255, 50);
	border: none;
	border-radius:3px;
	color: rgb(255, 255, 255);
}

QToolButton::menu-indicator { 
	image: None;
}

QToolButton:hover {
	border-radius:3px;
	color: rgb(255, 255, 255);
	background-color: rgb(21, 21, 21);
	border: none;
}

 
QToolButton:checked {
	border-radius:3px;
	color: rgb(255, 255, 255);
	background-color: rgb(21, 21, 21);
	border: none;
}</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_8">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="toolButton_2">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>OPPOSans B</family>
                 <pointsize>10</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>172.16.8.153:11100</string>
               </property>
               <property name="icon">
                <iconset resource="lib.qrc">
                 <normaloff>:/lib/node.png</normaloff>:/lib/node.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_4">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>OPPOSans B</family>
                 <pointsize>10</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>172.16.8.156:11100</string>
               </property>
               <property name="icon">
                <iconset resource="lib.qrc">
                 <normaloff>:/lib/node2.png</normaloff>:/lib/node2.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_5">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>50</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>OPPOSans B</family>
                 <pointsize>10</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>172.16.8.159:11100</string>
               </property>
               <property name="icon">
                <iconset resource="lib.qrc">
                 <normaloff>:/lib/node2.png</normaloff>:/lib/node2.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton_6">
               <property name="minimumSize">
                <size>
                 <width>180</width>
                 <height>50</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>180</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <family>OPPOSans B</family>
                 <pointsize>10</pointsize>
                 <stylestrategy>PreferAntialias</stylestrategy>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true"/>
               </property>
               <property name="text">
                <string>172.16.18.153:11100</string>
               </property>
               <property name="icon">
                <iconset resource="lib.qrc">
                 <normaloff>:/lib/node2.png</normaloff>:/lib/node2.png</iconset>
               </property>
               <property name="toolButtonStyle">
                <enum>Qt::ToolButtonTextBesideIcon</enum>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>309</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QStackedWidget" name="stackedWidget">
        <widget class="QWidget" name="page">
         <widget class="QWidget" name="horizontalWidget" native="true">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>0</y>
            <width>531</width>
            <height>181</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>209</height>
           </size>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="horizontalSpacing">
            <number>10</number>
           </property>
           <property name="verticalSpacing">
            <number>0</number>
           </property>
           <item row="2" column="3">
            <widget class="QWidget" name="widget_3" native="true">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>50</height>
              </size>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_save">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>OPPOSans B</family>
                  <pointsize>10</pointsize>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
}</string>
                </property>
                <property name="text">
                 <string>保存</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="toolButton_close">
                <property name="minimumSize">
                 <size>
                  <width>70</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>70</width>
                  <height>28</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <family>OPPOSans B</family>
                  <pointsize>10</pointsize>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">QToolButton {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
	border-radius: 5px;
}

QToolButton::menu-indicator { 
	image: None;
}

 
QToolButton:hover {
	color: rgb(255, 255, 255);
	background-color: rgb(255, 95, 95);
}</string>
                </property>
                <property name="text">
                 <string>取消</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_port_zk">
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>端口</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_host_zk">
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>主机</string>
             </property>
            </widget>
           </item>
           <item row="0" column="3">
            <widget class="QLineEdit" name="lineEdit_port_zk_data">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
             </property>
             <property name="text">
              <string>11100</string>
             </property>
             <property name="placeholderText">
              <string>22</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEdit_host_zk_data">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
             </property>
             <property name="text">
              <string>172.16.8.153</string>
             </property>
             <property name="placeholderText">
              <string>127.0.0.1</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_host_zk_2">
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>备注</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEdit_port_zk_data_2">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>28</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>OPPOSans B</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(25, 25, 25, 155);
border: 1px solid rgba(255, 255, 255, 0);
border-radius: 5px;
padding-left:10px;
padding-right:10px;</string>
             </property>
             <property name="text">
              <string>11100</string>
             </property>
             <property name="placeholderText">
              <string>22</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="lib.qrc"/>
 </resources>
 <connections/>
</ui>
